server/ServerFSM.cpp

```diff
@@ -506,6 +523,7 @@ sc::result MPLobby::react(const LobbyUpdate& msg) {
     // check if new lobby data changed player setup data.  if so, need to echo
     // back to sender with updated lobby details.
     bool player_setup_data_changed = (incoming_lobby_data.m_players != m_lobby_data->m_players);
+    bool is_drop_ready = false;
```

Comment:
`is_drop_ready` is unclear in this context, prefer (e.g.) `galaxy_setup_data_changed` or `lobby_data_changed`